web_search_autocomplete_prefetch/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Prefetch autocomplete offers
+============================
+
+When searching, the autocomplete options can be a bit frustrating because you
+will be offered choices that won't yield a result. This addon searches for the
+term in the background and only offers an option if this search has a result.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/162/8.0
```

Comment:
10.0 plz

Reply:
@JayVora-SerpentCS any other comments on this module?